services/api/src/typeDefs.js

```diff
@@ -278,17 +278,21 @@ const typeDefs = gql`
     uptimeRobotStatusPageId: String
   }
 
-  type Openshift {
+  type Kubernetes {
     id: Int
+    """
+    Unique Name
+    """
     name: String
-    consoleUrl: String
```

Comment:
I agree that we should rename this, but currently this removes it? let's call it like `apiUrl`?

Reply:
I thought we could get rid of it because the openshift/k8s services were going away, but forgot we still need it in `ssh` service at least. Renamed to `apiEndpoint`.